### ISSUE TITLE
Change votekick message

### ISSUE THREAD

**Issue by rekokoro** [Anuken/Mindustry-Suggestions#4755]
### Describe what you would like changed, and why.

First of all I don't care about commentators "lol, it's obvious", I'm sharing my opinion about what I'm actually observing in game.

Command in string "Type `/vote <y/n>` to vote" is misleading and people typing "y" or "n" in chat rather than full command.

I'm proposing to separate the line into two commands so it'll sound like "Type `/vote y` to agree or `/vote n` to disagree" so it easier to tell what does this command do.

### Describe the changes you want to propose. Include possible alternatives.

Another idea is to change voting process from text log (which is hard to follow on mobile) to notification. For that notification pop-up can be changed to hold two buttons  and dynamic text to reflect voting process. Same pop-up setting can be used for modifications like change map vote, surrender vote in pvp or custom gamemodes rather than fullscreen overlay.

**Comment by Plojoaze** [comment]
Moi personnellement j'aime bien ton idée et je suis tout a fais daccord avec.
